units/spawnhost_attach_volume.go

```diff
@@ -0,0 +1,101 @@
+package units
```

Comment:
Do these need to be jobs? If they don't take too long, I think it would be easier to have our APIs call AWS APIs directly so that the caller knows when it's done